VotingApplication/VotingApplication.Web/Scripts/Controllers/AddOptionDialogController.js

```diff
@@ -25,11 +25,17 @@
                 Description: form.description,
                 ChoiceNumber: null
             };
+            
+            $scope.choices.push(newChoice);
+
+            var formElement = angular.element(document.querySelector('form'));
```

Comment:
Perhaps we could be more specific here? I don't see us adding any more forms to the page but it feels like something that could cause problems if we forget about this.